src/resilience/exec_space/openMP/Resilient_OpenMP_Parallel_For.hpp

```diff
@@ -213,8 +213,8 @@ class ParallelFor< FunctorType
       const auto start{std::chrono::steady_clock::now()};
       KokkosResilience::inject_error_duplicates();
       const auto stop{std::chrono::steady_clock::now()};
-      KokkosResilience::ErrorTimerSettings::elapsed_seconds = KokkosResilience::ErrorTimerSettings::elapsed_seconds + (std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start));
-      KokkosResilience::ErrorTimerSettings::total_error_time = KokkosResilience::ErrorTimerSettings::total_error_time + KokkosResilience::ErrorTimerSettings::elapsed_seconds;
+      KokkosResilience::ErrorInjectionTracking::elapsed_seconds = KokkosResilience::ErrorInjectionTracking::elapsed_seconds + (std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start));
```

Comment:
Once ErrorInjectionTracking is a namespace, just throw a `using namespace KokkosResilience::ErrorInjectionTracking` at the top of this function and then throughout the function you can update as:
```suggestion
      elapsed_seconds += (std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start));
```